frontend/app/src/main.tsx

```diff
@@ -52,6 +53,10 @@ const router = createBrowserRouter([
 			},
 		],
 	},
+	{
+		path: "/test",
+		element: <HexagonalBoardBackground></HexagonalBoardBackground>,
+	},
```

Comment:
don't get me wrong, it's a nice page :D
but do we want to have it in main?

Reply:
absolutely right, added for testing purposes and forgot to remove